### PR TITLE
feat: Implement Dashboard Box "Sessions and Speakers"

### LINKED ISSUE #5940: Implement Dashboard Box "Sessions and Speakers"
The event dashboard does not have a dedicated box for sessions and speakers. Please implement a box.

In the box show info about
* number of speakers and include overview of pending, accepted etc. (all states)
* sessions and include an overview of pending, accepted etc. (all states)
* tracks, microlocations/rooms and session types
* when Call for speaker starts and ends

Screenshot from 2020-12-07 02-49-33

### PR DESCRIPTION
**Pull request by akash2408**
Fixes #5940 

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/components/events/view/overview/speaker-session.hbs [hunk 1]
```diff
@@ -0,0 +1,106 @@
+<div class="content">
+  <div class="header">{{t 'Sessions And Speakers'}}</div>
+</div>
+<div class="content">
+  <h4 class="ui header">{{t 'Summary'}}</h4>
+  <table class="ui very basic compact unstackable table">
+    <tbody>
+      <tr>
+        <td>{{t 'Session Types'}}</td>
```
Comment (daretobedifferent18) [inline app/templates/components/events/view/overview/speaker-session.hbs:9]: make this bold to make styling consistent

File: app/templates/components/events/view/overview/speaker-session.hbs [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            {{#each this.data.sessionTypes as |sessionType index|~}}
+              {{if (not-eq index 0) ','}} {{sessionType.name}}
+            {{~/each}}
+          {{else}}
+            {{t 'No Session Type present. '}} <a href="{{href-to 'events.view.edit.sessions-speakers'}}">{{t 'Click here to add'}}</a>
+          {{/if}}
+        </td>
+      </tr>
+      <tr>
+        <td>{{t 'Tracks'}}</td>
```
Comment (daretobedifferent18) [inline app/templates/components/events/view/overview/speaker-session.hbs:21]: here also

File: app/templates/components/events/view/overview/speaker-session.hbs [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+            {{#each this.data.tracks as |tracks index|~}}
+              {{if (not-eq index 0) ','}} {{tracks.name}}
+            {{~/each}}
+          {{else}}
+            {{t 'No Track Present. '}} <a href="{{href-to 'events.view.edit.sessions-speakers'}}">{{t 'Click here to add'}}</a>
+          {{/if}}
+        </td>
+      </tr>
+      <tr>
+        <td>{{t 'Microlocations'}}</td>
```
Comment (daretobedifferent18) [inline app/templates/components/events/view/overview/speaker-session.hbs:33]: here also

Comment (daretobedifferent18) [reply app/templates/components/events/view/overview/speaker-session.hbs:33]: Pls make it acc to other dashboard overview

File: app/templates/components/events/view/overview/speaker-session.hbs [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+      </tr>
+      <tr>
+        <td>{{t 'SpeakersCall'}}</td>
+        <td>
+          {{#if this.data.speakersCall}}
+            <strong>{{t 'Start Time'}}:</strong> {{general-date this.data.speakersCall.startsAt tz=this.data.event.timezone}}
+            <br>
+            <strong>{{t 'End Time'}}:</strong> {{general-date this.data.speakersCall.endsAt tz=this.data.event.timezone}}
+          {{else}}
+            {{t 'SpeakersCall not enabled. '}} <a href="{{href-to 'events.view.edit.sessions-speakers'}}">{{t 'Click here to add'}}</a>
```
Comment (daretobedifferent18) [inline app/templates/components/events/view/overview/speaker-session.hbs:52]: here also change to Call for Speakers

File: app/templates/components/events/view/overview/speaker-session.hbs [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+            {{#each this.data.microlocations as |microlocations index|~}}
+              {{if (not-eq index 0) ','}} {{microlocations.name}}
+            {{~/each}}
+          {{else}}
+            {{t 'No Microlocations Present. '}} <a href="{{href-to 'events.view.edit.sessions-speakers'}}">{{t 'Click here to add'}}</a>
+          {{/if}}
+        </td>
+      </tr>
+      <tr>
+        <td>{{t 'SpeakersCall'}}</td>
```
Comment (daretobedifferent18) [inline app/templates/components/events/view/overview/speaker-session.hbs:45]: also change this to Call for Speakers

File: app/components/events/view/overview/speaker-session.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
+import classic from 'ember-classic-decorator';
+import { classNames } from '@ember-decorators/component';
+import { computed } from '@ember/object';
+import Component from '@ember/component';
+
+@classic
+@classNames('ui', 'fluid', 'card')
+export default class SpeakerSession extends Component {
+  @computed('data.orderStat.tickets')
+  get tickets() {
```
Comment (daretobedifferent18) [inline app/components/events/view/overview/speaker-session.js:10]: what is the use of all these computed prop. Where are they used?

Comment (akash2408) [reply app/components/events/view/overview/speaker-session.js:10]: Sorry i did this by mistake
and i ll do above changes.thanks for the review

Comment (daretobedifferent18) [reply app/components/events/view/overview/speaker-session.js:10]: no need of this file.
Delete it

Comment (daretobedifferent18) [reply app/components/events/view/overview/speaker-session.js:10]: And also make sure that it is working good in mobile view

File: app/templates/components/events/view/overview/speaker-session.hbs [hunk 7]
```diff
@@ -0,0 +1,106 @@
+<div class="content">
+  <div class="header">{{t 'Sessions And Speakers'}}</div>
+</div>
+<div class="content">
+  <h3 class="ui header">{{t 'Summary'}}</h3>
```
Comment (daretobedifferent18) [inline app/templates/components/events/view/overview/speaker-session.hbs:5]: change back to h4

**Comment by mariobehling** [comment]
Thanks!

Please also make the following changes:
* Keep No. of Speakers and No. of Sessions | 1 in the "Overview" box
* Do not make "Start Time/End Time" bold
* Move box above box "Team Members"

**Comment by mariobehling** [comment]
Change "Sessions And Speakers" to "Sessions and Speakers"

**Comment by mariobehling** [comment]
Let's try to also keep details correct, e.g. after ":" there is always a space.

Also change the date to standard implementation for long dates (if it was not correct in the general box previously it should be correct in the new box. We adapted two formats recently). Dates should have this kind of format:
- Sunday, 6 December, 2020 9:00 AM (+07)
- Sunday, 13 December, 2020 11:59 PM (+07)

**Comment by akash2408** [comment]
@iamareebjamal  please review this and let me know for any changes?

File: app/templates/components/events/view/overview/speaker-session.hbs [hunk 8]
```diff
@@ -0,0 +1,116 @@
+<div class="content">
+  <div class="header">{{t 'Sessions And Speakers'}}</div>
```
Comment (maze-runnar) [inline app/templates/components/events/view/overview/speaker-session.hbs:2]: Sessions ans Speakers

Comment (daretobedifferent18) [reply app/templates/components/events/view/overview/speaker-session.hbs:2]: Only do this change, everything else is working fine.

**Review by iamareebjamal — APPROVED** [review]

**Comment by mariobehling** [comment]
Thank you! The box should only show up if session and speakers wizard is activated. I added a follow up issue here https://github.com/fossasia/open-event-frontend/issues/6033

**Pull request merged**